testing/spark_temp/simple_test.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
```

Comment:
What is this script for? Can you add a comment please?

Reply:
This was for local testing, I can remove it.